tools/topology/sof-imx8-processing-wm8960.m4

```diff
@@ -0,0 +1,91 @@
+#
+# Topology for i.MX8MP board with wm8960 codec
```

Comment:
If this topology is for 8QM and 8QXP maybe you should remove/update the i.MX8MP comment.

Reply:
Yes, I think that's a good idea. Copy paste error :).